homeassistant/components/mqtt/strings.json

```
@@ -1,5 +1,9 @@
 {
   "issues": {
+    "deprecated_vacuum_battery_feature": {
+      "title": "Deprecated battery feature used",
+      "description": "Vacuum entity {entity_id} implements the battery feature which is deprecated. This will stop working in Home Assistant 2026.3. Implement a separate entity for the battery status instead. To fix the issue `battery` feature should be removed from the configured supported features, and Home Assistant should be restarted to fix this issue."
```

- I assume this should be "battery state" here, not "status"?
- Adding a "the" makes the last sentence much more readable.
Otherwise it's "… the issue `battery` feature …" here.
- As "battery" is quoted in backticks in the last sentence this should perhaps be done in the first, too.

```suggestion
      "description": "Vacuum entity {entity_id} implements the `battery` feature which is deprecated. This will stop working in Home Assistant 2026.3. Implement a separate entity for the battery state instead. To fix the issue the `battery` feature should be removed from the configured supported features, and Home Assistant should be restarted to fix this issue."
```